packages/build/tests/core/tests.js

```diff
@@ -361,6 +361,18 @@ test('Functions config is passed to zip-it-and-ship-it (3)', async (t) => {
   await runFixture(t, 'functions_config_3')
 })
 
+test('Functions schedule is parsed (1)', async (t) => {
+  await runFixture(t, 'functions_schedule_1')
+})
+
+test('Functions schedule is parsed (2)', async (t) => {
+  await runFixture(t, 'functions_schedule_2')
+})
+
+test('Functions schedule is parsed (3)', async (t) => {
```

Comment:
Should we add more descriptive names (as opposed to the integer count)?

Reply:
https://github.com/netlify/build/pull/3631/commits/2d52d883368f36e04d91a63ce1990f00d896d1b0